app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,73 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        fetch_authenticator_secrets: Authentication::Util::FetchAuthenticatorSecrets.new,
+        discover_identity_provider: Authentication::OAuth::DiscoverIdentityProvider.new,
+        create_signing_key: Authentication::AuthnJwt::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::FetchIssuerValue.new,
+        validate_uri_parameters: Authentication::AuthnJwt::ValidateUriBasedParameters.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityFromDecodedTokenProvider
```

Comment:
please match the param name to the class name. it should be `identity_from_decoded_token_provider_class` (the `_class` is there as this is a class and not an instance)